meshtastic/node.py

```
@@ -587,14 +641,36 @@ def writeConfig(self, config_name: str) -> None:
         config_setter = getattr(p, setter_name)
         getattr(config_setter, config_name).CopyFrom(source_config)
 
-        logger.debug(f"Wrote: {config_name}")
+        logger.debug("Wrote: %s", config_name)
         if self == self.iface.localNode:
             onResponse = None
         else:
             onResponse = self.onAckNak
```

Ruff (0.15.5)

[warning] Use ternary operator `onResponse = None if self == self.iface.localNode else self.onAckNak` instead of `if`-`else`-block

Replace `if`-`else`-block with `onResponse = None if self == self.iface.localNode else self.onAckNak`

(SIM108)